src/config/config.ts

```diff
@@ -110,10 +110,8 @@ const submissionsTopUp = parseInt(process.env.SUBMISSIONS_TOP_UP, 10) || 0
 
 /**
  * Content Security Policy reporting
- * HelmetJS reportUri param requires non-empty string, so 'undefined' is
- * declared.
  */
-const cspReportUri = process.env.CSP_REPORT_URI || 'undefined'
```

Comment:
Might need to rebase and fix merge issues after https://github.com/opengovsg/FormSG/pull/190 is merged in